changelog.d/4642.bugfix

```diff
@@ -0,0 +1 @@
+Transfer bans on room upgrade.
```

Comment:
I call it a feature rather than a bugfix.

Reply:
Really? All the other room upgrade stuff I've been doing have been bug fixes.

Reply:
I'd argue that's incorrect, then. This is a new feature for room upgrades as far as I'm concerned.

Reply:
Kk, I have no strong feelings one way or the other.